src/systems/physics/Physics.hh

```diff
@@ -39,6 +39,7 @@
 #include <ignition/physics/RemoveEntities.hh>
 #include <ignition/physics/Shape.hh>
 #include <ignition/physics/SphereShape.hh>
+#include <ignition/physics/World.hh>
```

Comment:
Is there a reason for including this header (and other headers) in the header file, if they're really only used in the source file? Does it have something to do with making builds green for macOS and/or Windows?

Reply:
There's a little comment on line 26, these were originally here to satisfy the initial `entityCast` implementation. There's a chance that this isn't needed anymore since #586, I haven't checked.